app/controllers/hotels_controller.rb

```diff
@@ -1,12 +1,11 @@
-class HotelsController < ApplicationController  
+class HotelsController < ApplicationController
+  before_filter :authenticate_user!
+  load_and_authorize_resource
   before_action :set_city, only: [:edit, :new, :create,:update]
-  before_action :set_hotel, only: [:show, :edit, :update, :destroy]
+  before_action :set_hotel, only: [:show, :edit, :update, :destroy, :auth_hotel]
```

Comment:
because to authorized hotel(update hotel status from pending to approved) i need a reference of hotel